src/components/tutorials-landing-view/certification-content-card-link/index.tsx

```diff
@@ -6,15 +6,10 @@
 import { trackCertificationCardLinkClicked } from 'views/tutorials-landing/analytics'
 import { type CertificationContentCardLinkProps } from '../types'
 import ContentCardLink from '../content-card-link'
-import networkingAutomationGraphic from './img/consul.svg'
 import infrastructionAutomationGraphic from './img/terraform.svg'
 import securityAutomationGraphic from './img/vault.svg'
 
 const CERTIFICATION_PROGRAM_SLUGS_TO_BACKGROUND_IMAGES = {
-	'networking-automation': {
-		url: networkingAutomationGraphic,
-		lightOrDark: 'dark',
-	},
```

Comment:
Stopped importing and using the image here since this slug shouldn't appear anymore. 
Can potentially delete the entire `svg` deleted in a follow up PR since it isn't used anywhere else.

Reply:
Yes, I think as long as you're 100% sure that `svg` is not in use, it can be deleted 👍